app/components/forms/admin/settings/payment-gateway-form.js

```diff
@@ -213,6 +257,14 @@ export default Component.extend(FormMixin, {
             'omiseLiveSecret' : null
           });
         }
+        if (!this.isCheckedPaytm) {
```

Comment:
Why is this needed, this will set all properties to null once I disable payTM for a short time and also `is_payTM_activated` boolean would be enough for this.